src/uploader.rs

```diff
@@ -59,6 +62,10 @@ pub struct ConexUploader {
 }
 
 const UPLOAD_CHUNK_SIZE: usize = 512 * 1024 * 1024;
+/* @ex 
+    key: layer_id
+    files: list of ConextFiles within key layer 
+*/
```

Comment:
let's follow rust's function documentation guide https://stackoverflow.com/questions/30009650/how-to-document-rust-function-arguments